cmd/build/v2/build.go

```diff
@@ -65,6 +65,7 @@ type oktetoBuilderConfigInterface interface {
 	IsOkteto() bool
 	GetAnonymizedRepo() string
 	GetBuildContextHash(*model.BuildInfo) string
+	IsSmartBuildsEnable() bool
 }
```

Comment:
Should we call it `IsSmartBuildsEnabled`?

Reply:
36042266fd89ed3e43ee7d21db2803a63c36a43e